instrumentation/elasticsearch/elasticsearch-api-client-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/apiclient/v8_0/ElasticsearchEndpointMap.java

```diff
@@ -0,0 +1,840 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.elasticsearch.apiclient.v8_0;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class ElasticsearchEndpointMap {
+  private static final Set<String> searchEndpoints =
+      new HashSet<>(
+          Arrays.asList(
+              "search",
+              "async_search.submit",
+              "msearch",
+              "eql.search",
+              "terms_enum",
+              "search_template",
+              "msearch_template",
+              "render_search_template"));
+
+  private static volatile Map<String, String[]> routesMap;
+
+  private ElasticsearchEndpointMap() {}
+
+  public static boolean isSearchEndpoint(String endpointId) {
```

Comment:
this method doesn't appear to be used

Reply:
For java instrumentations we have this far required sanitization of sensitive info in default configuration.

Reply:
sanitization of ES queries seems to be very tricky, made it an opt-in config option for now